app/client/src/components/shared/HmwMap/index.js

```diff
@@ -0,0 +1,79 @@
+// @flow
+
+import React from 'react';
+import type { Node } from 'react';
+import { css } from 'styled-components/macro';
+import Map from '@arcgis/core/Map';
+import MapView from '@arcgis/core/views/MapView';
+import MapWidgets from 'components/shared/MapWidgets';
+import MapMouseEvents from 'components/shared/MapMouseEvents';
+// contexts
+import { LocationSearchContext } from 'contexts/locationSearch';
+
+// --- styled components ---
+const mapContainerStyles = css`
+  position: absolute;
+  width: 100%;
+  height: 100%;
+`;
+
+// --- components ---
+type Props = {
+  layers: Object,
+  startingExtent?: Object,
+  children?: Node,
+};
+
+function HmwMap({ layers = null, startingExtent = null, children }: Props) {
```

Comment:
One more thought on this file... I'm going to rename this component to be just `Map` so it's organized alongside our related mapping components (e.g. `MapLegend`, `MapLoadingSpinner`, `MapMouseEvents`, etc.)

I'll just import the default module as a different name for use within this file:
```diff
- import Map from '@arcgis/core/Map';
+ import EsriMap from '@arcgis/core/Map';
```